mod_reforged/hooks/config/strings.nut

```diff
@@ -1731,7 +1732,32 @@ foreach (vanillaDesc in vanillaDescriptions)
 		}]
 	}),
 	RF_RealizedPotential = ::UPD.getDescription({
-		Fluff = "From bones to brawn! This character has truly come a long way. Who was once a dreg of society is now a full-fledged mercenary. " + ::MSU.Text.colorPositive("\n\nAll perk points have been refunded and attributes increased."),
+		Fluff = "From bones to brawn! This character has truly come a long way. Who was once a dreg of society is now a full-fledged mercenary.",
+		Effects = [
+			{
+				Type = ::UPD.EffectType.Passive,
+				Description = [
+					"Require " + ::MSU.Text.colorNegative("100%") + " more daily wage.",
+				]
+			},
+			{
+				Type = ::UPD.EffectType.OneTimeEffect,
+				Description = [
```

Comment:
As part of this change, the logic inside `onUpdateLevel` of `perk_rf_promised_potential` must be moved to `onAdded` of `perk_rf_realized_potential` with the `if (!this.m.IsNew)` check. Then all the promised potential perk does is add the realized potential perk and the latter handles all the changes to the character, as described in its tooltip.